src/components/layout/Layout.module.scss

```
@@ -1,12 +1,14 @@
 @use 'theme';
 
+$height: var(--height, '100vh');
```

The browser would only use the fallback variable just in case the variable is not defined/exist, it will not use fallback if the variable is invalid. So the default better uses in the `*.tsx` file.
```suggestion
$height: var(--height);
```